misk/src/test/kotlin/misk/cloud/gcp/security/keys/GcpKeyServiceTest.kt

```diff
@@ -46,13 +46,17 @@ internal class GcpKeyServiceTest {
         assertThat(plainText.utf8()).isEqualTo("decrypted")
     }
 
-    @Test(expected = IllegalArgumentException::class)
+    @Test
     fun invalidEncryptKey() {
-        keyManager.encrypt("unknown_key", ByteString.encodeUtf8("should fail"))
+        assertThrows(IllegalArgumentException::class.java) {
```

Comment:
This file seems to only have Junit 4->5 changes. Should probably be in a separate PR?

Reply:
Yeah will pull into their own PR

Reply:
These are actually manik's changes from another branch, not sure why this PR is picking up rebases.